package.json

```diff
@@ -66,9 +69,11 @@
   },
   "license": "MIT",
   "scripts": {
-    "postinstall": "bower install && ./tasks/privateDependencies.sh",
```

Comment:
Why did you remove this hook ?

Reply:
two reasons. One, bower was not happy running inside npm inside a container. With the `--allow-root` option, it works.

The second reason is harder to overcome. The privateDependencies script uses git to clone stuff from github. When running inside the node docker image, that won't work. My change here was to move it to the Makefile, and run it directly. I'd be happy to find some alternative that works both ways. For example, the script could exit early if the dependencies seem to already be there.

Speaking of bower, are there any plans to remove it? In their official repo, they recommend "yarn and webpack for new front-end projects". I'm sure npm@5 and webpack would do just fine.

Reply:
> For example, the script could exit early if the dependencies seem to already be there.

The goal of this script is to always install a new version. We don't want to do nothing if there is already a directiry inside vendor, that's why there is a `$ rm -r`

I hope to kill bower in a few month, today it works and that's not something we need to improve yet.

Reply:
cool. That makes sense. I suppose always cloning is more straight forward then cloning or updating-if-already-cloned.